Travel Scheduler/View Controllers/PlaceWebsiteViewController.m

```diff
@@ -0,0 +1,45 @@
+//
+//  PlaceWebsiteViewController.m
+//  Travel Scheduler
+//
+//  Created by gilemos on 7/31/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "PlaceWebsiteViewController.h"
+
+@interface PlaceWebsiteViewController ()
+
+@end
+
+@implementation PlaceWebsiteViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    NSURL *url = [NSURL URLWithString:self.websiteURL];
+    NSURLRequest *request = [NSURLRequest requestWithURL:url];
+    
+    _webView = [[WKWebView alloc] initWithFrame:self.view.frame];
+    [_webView loadRequest:request];
+    _webView.frame = CGRectMake(self.view.frame.origin.x,self.view.frame.origin.y, self.view.frame.size.width, self.view.frame.size.height);
```

Comment:
You should probably put this line into the viewWillLayoutSubviews method so that it autolayouts. Also, if you just want the webView frame to take up the entire view controller, I think you can just say _webView.frame = self.view.bounds or self.view.frame.